contract-builder/run_m1.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+HOST_DIR="${HOST_DIR:-$(pwd)/..}"
+
+docker run \
+     --platform linux/arm64 \
```

Comment:
We should add `--platform linux/arm64` to `examples/docker_build.sh` as well. But I think we would have to publish an `arm64` docker image to our Dockerhub. Not sure who manages it though.

CC @austinabell @ChaoticTempest

Reply:
related @rtsai123 @anshal-savla might be worth considering if updating the docker image.